src/wallet/wallet.h

```diff
@@ -855,6 +855,7 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
     //! Load metadata (used by LoadWallet)
     void LoadKeyMetadata(const CKeyID& keyID, const CKeyMetadata &metadata) EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
     void LoadScriptMetadata(const CScriptID& script_id, const CKeyMetadata &metadata) EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
+    void UpgradeKeyMetadata();
```

Comment:
This needs a description of what it's doing.

Reply:
Done